src/main/webapp/help.html

```diff
@@ -6,17 +6,18 @@
     <link rel="stylesheet" href="assets/css/custom_style.css">
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <title>Help for Sections Manager</title>
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
+    <script type="text/javascript">
+	$('document').ready(function() {
+		 $("#buildInfo").load("build.txt");
```

Comment:
What happens if the call to load() fails.  You should check for errors.  For examples, see src/main/webapps/about.html and src/main/webapps/index.html in GDrive.  Those use a callback to either display the contents of build.txt or an error message, then it calls show() on #appInfo, which had been hidden previously.